byron/ledger/executable-spec/test/Ledger/Delegation/Examples.hs

```diff
@@ -38,18 +39,18 @@ deleg =
   [ testGroup "Activation"
     [ testCase "Example 0" $ checkTrace @ADELEG genKeys $
 
-      pure (DState [] [])
+      pure (DState (Bimap.fromList []) [])
 
-      .- (s 0, (gk 0, k 10)) .-> DState [(gk 0, k 10)]
+      .- (s 0, (gk 0, k 10)) .-> DState (Bimap.fromList [(gk 0, k 10)])
```

Comment:
Oh, it seems `Bimap` does not have a `IsList` instance. Would it make sense to leave this for now, and make an issue about sending a PR against [this repo](https://github.com/joelwilliamson/bimap) to add the `IsList` instance and change this back? In the case of the test examples I am a bit worried about keeping them readable :)

Reply:
Good idea. I can look at raising a PR upstream for this.

Reply:
https://github.com/joelwilliamson/bimap/pull/10

Reply:
Nice! :tada: